data/templates/css/wikimedia.scss

```diff
@@ -630,16 +630,16 @@ and (max-device-width: 719px) {
         }
     }
 
-    .mw-parser-output {
+    #bodycontents #inside-content div {
```

Comment:
If we got time to test, we should try using first-of-type on div element so we target only first div.
With current selector every div will have flex display attribute

_#bodycontents #inside-content div:first-of-type {_

Reply:
That works too! Updated the PR.